cmd/crc/cmd/status.go

```diff
@@ -180,12 +180,15 @@ func (s *status) prettyPrintTo(writer io.Writer) error {
 		{"CRC VM", s.CrcStatus},
 	}
 
-	if s.OpenShiftVersion != "" {
+	if s.Preset == preset.OpenShift {
 		lines = append(lines, line{"OpenShift", openshiftStatus(s)})
 	}
-	if s.PodmanVersion != "" {
+	if s.Preset == preset.Podman {
```

Comment:
With openshift/microshift, we also expose podman, so I think we always want to print this if podman version is set.

Reply:
Nobody can know this except you and Gerard. Please split this behaviour change to its own commit with a proper rationale so that it's obvious to everyone that the change is intentional, and why it's being done.